build.gradle

```diff
@@ -52,7 +52,19 @@ semanticRelease {
 }
 
 task integTest {
-    dependsOn test
+    dependsOn cleanIntegTest, test
+}
+
+cleanIntegTest {
+  delete new File(buildDir, 'nebulatest')
```

Comment:
The problem with this line is, that it is executed during configuration time, i.e. every time you execute any gradle task it is executed when the build file is parsed.

What you want, is

```
task cleanIntegTest {
    doLast {
        delete new File(buildDir, 'nebulatest')
    }
}
```
In this case, however, it would suffice to define `cleanTest`, since `integTest` is just a convenience "task" to also execute the integ tests when running the tests.

But you could also get this behavior by defining

```
test {
    outputs.dir new File(buildDir, 'nebulatest')
    dependsOn cleanTest
}
```

Reply:
Why then you merged incorrect fix? ))

According to my investigation this doesn't happen. `cleanIntegTest` is an instance of `Delete` task, and `delete` is just a DSL to add files and dirs for deletion.

I see there is another problem. Deletion should happen before `test` only. `cleanIntegTest` is not needed. That wasn't planned, it was rebase error.